### ISSUE TITLE
Translated content missing

### ISSUE THREAD

**Issue by ryanlewis26** [nathanwoulfe/Plumber#145]
**Describe the bug**
We're experiencing missing content on some translations for one of our sites.

In both examples below the the content for each is present in the .xlf file (attached) but the system isn’t reading them for some reason.

.xlf examples: 
[XLF_Examples.zip](https://github.com/nathanwoulfe/Plumber/files/5590923/XLF_Examples.zip)

1. ENG to FR, key features block not working:
- in this case the translations are showing in Translation Manager when submitted:
![image](https://user-images.githubusercontent.com/74975258/100114311-30cb3680-2e69-11eb-953a-2b226408ad76.png)
- but when checking the content page after publishing the translations are missing
![image](https://user-images.githubusercontent.com/74975258/100114410-50faf580-2e69-11eb-9976-ea1bd820e5e0.png)

2. ENG to DE, two column block not working:
- in this case the translations are returned blank in Translation Manager:
![image](https://user-images.githubusercontent.com/74975258/100114785-afc06f00-2e69-11eb-963d-fff0d8bc9390.png)

**Expected behavior**
When the translation is submitted the information in the .xlf files is accepted and appears both in Translation Manager and on the translated page

**Desktop (please complete the following information):**
 - Windows 10
 - Browser - Chrome (latest)

**Plumber version**
2.4.12.0

**Comment by nathanwoulfe** [comment]
Hi @ryanlewis26, I'm not sure how this would be related to Plumber - Plumber simply uses the ContentService to publish nodes at the completion of the workflow, there's no modification of the content. 

If Translation Manager performs some other magic on publish (by hooking content events), this should still work, unless those events are triggered from the client side, and therefore not raised by Plumber on the server. 

@KevinJump might be able to provide some insights...

**Comment by KevinJump** [comment]
yes - this is a Translation Manager issue. looks like we had a problem with Repeatable strings inside nested content/grids. But we have a fix now, it will be in the next release - either Friday or early next week.

**Comment by nathanwoulfe** [comment]
Magic! I love issues that don't mean work!

**Comment by ryanlewis26** [comment]
@KevinJump @nathanwoulfe Fantastic! Thanks both for your time.

**Comment by ryanlewis26** [comment]
Hi @KevinJump - do you know when this new release will be available? Just keen to get upgraded to fix this issue.

**Comment by KevinJump** [comment]
yeah just published it now ... waiting for nuget to replicate.

**Comment by KevinJump** [comment]
you should be able to grab it from here : https://www.nuget.org/packages/Jumoo.TranslationManager/8.4.2

**Comment by ryanlewis26** [comment]
ah apologies! I thought I added to my initial post but we're not upgraded to umbraco 8 yet. We're on Umbraco v7.14.0 using Translation Manager v2.4.12.0
will Umbraco v7 be supported with this fix?

**Comment by KevinJump** [comment]
ahh i will have to go check. sorry, thought it was v8

**Comment by KevinJump** [comment]
https://www.nuget.org/packages/Jumoo.TranslationManager/2.4.13
